docs/user-tutorial/data-diagnosis.md

```diff
@@ -0,0 +1,144 @@
+---
+id: data-diagnosis
+---
+
+# Data Diagnosis
+
+## Introduction
+
+This tool is to filter the defective machines from thousands of benchmarking results automatically according to rules defined in **Rule file**.
+
+## Input
+
+The input mainly includes 2 files:
+
+ - **raw data**: jsonl file including multiple nodes' results automatically generated by SuperBench runner.
+
+    `Tips`: this file can be found at ${output-dir}/results-summary.jsonl after each successful run.
+
+ - **Rule file**: It uses YAML format and includes each metrics' rules to filter defective servers for diagnosis.
```

Comment:
`defective servers` or `defective machines` ? Please unify the term.

Reply:
updated, unified to defective machines